modules/standard/Path.chpl

```diff
@@ -778,39 +715,40 @@ proc relPath(path: string, start:string=curDir): string throws {
   return joinPath(outComps.toArray());
 }
 
-pragma "no doc"
-pragma "last resort"
-proc relPath(name: string, start:string=curDir): string throws {
-  compilerWarning("Path.relPath: Argument 'name' is deprecated - use 'path' instead");
-  return relPath(path=name,start);
-}
-
 /*
-  Returns a relative filepath to the path in this :type:`~IO.file` either from
-  the current directory or an optional `start` directory. The filesystem is not
-  accessed to verify the existence of the named path or the specified starting
-  location.
+  Returns a relative filepath to the path in a given :type:`~IO.file` either
+  from the current directory or an optional `start` directory. The filesystem is
+  not accessed to verify the existence of the named path or the specified
+  starting location.
 
   .. warning::
 
     This method is unsafe for use in a parallel environment due to its
     reliance on :proc:`~FileSystem.locale.cwd()`. Another task on the current
     locale may change the current working directory at any time.
 
+  :arg f: The file
+  :type f: :type:`~IO.file`
+
   :arg start: The location from which access to the path in this
     :type:`~IO.file` is desired. If no value is provided, defaults to
     :const:`curDir`.
   :type start: `string`
 
-  :return: The relative filepath to the path in this :type:`~IO.file`.
+  :return: The relative filepath to the path in the given :type:`~IO.file`.
   :rtype: `string`
 
   :throws SystemError: Upon failure to get the current working directory.
 */
-proc file.relPath(start:string=curDir): string throws {
-  import Path;
+proc relPath(f: file, start:string=curDir): string throws {
   // Have to prefix module name to avoid muddying name resolution.
-  return Path.relPath(this.path, start);
+  return Path.relPath(f.path, start);
```

Comment:
I think the `Path.` portion should now be unnecessary, since you aren't in a method any more.  There's probably other places where this applies that I've already passed and didn't notice